README.md

```diff
@@ -294,7 +216,8 @@ dependencyRecommendations {
 }
 ```
 
-### Copyright Checks
+Adds the following tasks:
 
-By default Baseline enforces Palantir copyright at the beginning of files. To change this, edit the template copyright
-in `.baseline/copyright/*.txt` and the RegexpHeader checkstyle configuration in `.baseline/checkstyle/checkstyle.xml`
+- `checkVersionsProps` - A catch-all task to lint your versions.props file.
+- `checkBomConflict` - Ensures your versions.props don't conflict with versions sourced from a BOM.
```

Comment:
can we say, ... "your versions.props pins don't force the same version that is already recommended by a BOM"?